vars/govuk.groovy

```diff
@@ -552,33 +552,51 @@ def setEnvGitCommit() {
 /**
  * Runs the ruby linter. Only lint commits that are not in master.
  */
-def rubyLinter(String dirs = 'app spec lib', boolean lintDiff = true) {
+def rubyLinter(String repoName, String dirs = 'app spec lib', boolean lintDiff = true) {
   setEnvGitCommit()
+
   if (!isCurrentCommitOnMaster()) {
     echo 'Running Ruby linter'
 
     withStatsdTiming("ruby_lint") {
-      sh("bundle exec govuk-lint-ruby \
+      def lintIsOkay = runsSuccessfully("bundle exec govuk-lint-ruby \
          --parallel \
          ${lintDiff ? '--diff --cached' : ''} \
          --format html --out rubocop-${GIT_COMMIT}.html \
          --format clang \
          ${dirs}"
       )
+
+      if (lintIsOkay) {
+        setBuildStatus("ruby-linter", getFullCommitHash(), "No Ruby linting issues found", "SUCCESS", repoName)
+      } else {
+        setBuildStatus("ruby-linter", getFullCommitHash(), "Ruby linter found issues", "FAILURE", repoName)
+      }
     }
   }
 }
 
 /**
  * Runs the SASS linter
  */
-def sassLinter(String dirs = 'app/assets/stylesheets') {
+def sassLinter(String dirs = 'app/assets/stylesheets', String repoName) {
   echo 'Running SASS linter'
   withStatsdTiming("sass_lint") {
-    sh("bundle exec govuk-lint-sass ${dirs}")
+    if (runsSuccessfully("bundle exec govuk-lint-sass ${dirs}")) {
+      setBuildStatus("css-linter", getFullCommitHash(), "No CSS linting issues found", "SUCCESS", repoName)
+    } else {
+      setBuildStatus("css-linter", getFullCommitHash(), "CSS linter found issues", "FAILURE", repoName)
```

Comment:
Given this is SASS, not CSS, maybe change the above to be more consistent?

Reply:
I think our initial naming of that job was wrong. We're actually linting `scss`, because `sass` is [the deprecated syntax that relied on indentation](https://responsivedesign.is/articles/difference-between-sass-and-scss/). I was thinking to name it css-lint because it also covers non-scss CSS.

Reply:
Ok, that makes sense :+1: